src/Command/TestCommands.php

```diff
@@ -90,4 +91,74 @@ public function testPhpunit(array $args)
         }
         return $phpunitExec->run();
     }
+
+    /**
+     * Proxy to phpcs.
+     */
+    public function phpcs(array $args) {
+        $dktl_dir = Util::getDktlDirectory();
+        $phpcs_command = "{$dktl_dir}/vendor/bin/phpcs";
+
+        $task = $this->taskExec("{$phpcs_command} --config-set installed_paths {$dktl_dir}/vendor/drupal/coder/coder_sniffer");
+        $task->run();
+
+        $task = $this->taskExec($phpcs_command);
+        foreach ($args as $arg) {
+            $task->arg($arg);
+        }
+        $task->run();
+    }
+
+    /**
+     * Proxy to phpcbf.
+     */
+    public function phpcbf(array $args) {
+        $dktl_dir = Util::getDktlDirectory();
+        $phpcs_command = "{$dktl_dir}/vendor/bin/phpcbf";
+
+        $task = $this->taskExec($phpcs_command);
+        foreach ($args as $arg) {
+            $task->arg($arg);
+        }
+        $task->run();
+    }
+
+    /**
+     * Preconfigured linting for paths inside of the repo.
+     */
+    public function testLint(array $paths) {
+        $dktl_dir = Util::getDktlDirectory();
+        $project_dir = Util::getProjectDirectory();
+
+        $phpcs_command = "{$dktl_dir}/vendor/bin/phpcs";
```

Comment:
Maybe create a class property or get method for phpcs_command so we don't have this redundant code?

Reply:
...although actually, wouldn't it make more sense to re-use `$this->phpcs`?